modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java

```diff
@@ -26,7 +26,7 @@
 
 public class OpenIdCredentialRefreshable extends WrappingOpenIdCredential
 {
-    private static final Logger LOG =
+    private static final Logger logger =
```

Comment:
That's the point. You see, that we are already confused, so don't make it worse :). If you want to enforce a single style, then you can do it with a dedicated commit and describe there why.